File: part 4/practice/Token.py
class Token(object):
    PLUS, MINUS, MULTIPLY, DIVIDE = 'PLUS', 'MINUS', 'MULTIPLY', 'DIVIDE'
    INTEGER = 'INTEGER'
    WHITESPACE, EOF = 'WHITESPACE', 'EOF'

    def __init__(self, type, value):
        self.type = type
        self.value = value

    def __str__(self):
        """String representation of the class instance.

        Examples:
            Token(INTEGER, 3)
            Token(PLUS, '+')
        """
        return 'Token({type}, {value})'.format(
            type=self.type,
            value=repr(self.value),
        )

    def __repr__(self):
        return self.__str__()

File: part 4/practice/Token.cs
using System;

namespace test.part_4.practice
{
    public class Token
    {
        public static string PLUS = "PLUS";
        public static string MINUS = "MINUS";
        public static string MULTIPLY = "MULTIPLY";
        public static string DIVIDE = "DIVIDE";
        public static string INT = "INT";

        public string Type { get; set; }
        public string Value { get; set; }

        public Token(string type, string value)
        {
            this.Type = type;
            this.Value = value;
        }

        public override string ToString()
        {
            return $"Type: {Type}, Value: {Value}";
        }
    }
}
